packages/documentation/src/stories/components/tooltip/tooltip.stories.ts

```diff
@@ -75,6 +75,7 @@ function render(args: Args) {
       id="${args.id}"
       class="hydrated bg-${args.backgroundColor}"
       placement="${ifDefined(args.placement)}"
+      arrow="${ifDefined(args.arrow)}"
```

Comment:
You should import the `nothing` object from lit, set `args.arrow` to `false` in the meta object and then define the arrow binding like this:
`arrow="${args.arrow ? args.arrow : nothing}"`
This will make the arrowbinding appear only if `arrow` is set to `true` and show the false/true toggler in the storybook control table from the very beginning.